### PR TITLE
Added more information on usage

### PR DESCRIPTION
**Pull request by dougtoppin**
I added some more information about subscriptions, entitlements and some cost information if you think it is helpful to include.
You might consider an option to only run 1 master, 1 node and 1 etcd to reduce the number of entitlements required and reduce the instance cost.
I think that full HA is a good thing to have as a reference on how to do it but also being able to run your own OpenShift environment for learning/testing is very helpful (and affordable).

### CODE REVIEW

**Comment by ngerasimatos** [comment]
@dougtoppin The next release of OCP will not officially support 1 master, infra node and app node. SInce we are trying to keep the design as close to reference architecture as possible these changes won't be possible :(

As a side note, we are also removing the option to run 1/1/1 in our Azure Marketplace offering as well.

**Comment by dougtoppin** [comment]
Did not know that, still learning.
tks.

**Comment by ngerasimatos** [comment]
@dougtoppin No problem, with as fast as all of this is emerging we are all learning on the fly ;)

File: LICENSE.txt [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright {2017} {Doug Toppin}
```
Comment (avattathil) [inline LICENSE.txt]: **The LICENSE file cannot have any modifications.** It will not pass our legal review
In the future please don't edit this file.

**Comment by avattathil** [comment]
Please resubmit without copyright mods

**Pull request closed without merging**